docs/guide/security_checklist.md

```diff
@@ -4,65 +4,192 @@ description: Ensure that your Ibexa DXP installation is secure by following our
 
 # Security checklist
 
-When getting ready to go live with your project for the first time, or when re-launching it,
-make sure that your setup is secure.
+When getting ready to go live with your project for the first time, or when re-launching it, make sure that your setup is secure.
 
 !!! caution
 
-    Security is an ongoing process. After going live, you should pay attention to security advisories
-    released via [your service portal,](https://support.ibexa.co/)
-    or via [Security advisories](https://developers.ibexa.co/security-advisories) if you're not a subscriber.
-    
+    Security is an ongoing process. After going live, you should pay attention to security advisories released via [your service portal](https://support.ibexa.co/), or via [Security advisories](https://developers.ibexa.co/security-advisories) if you're not a subscriber.
+
+## [[= product_name =]]
+
+### Carefully select admin users
+
+Make sure Admin users and other privileged users who have access to System Information and setup in the back end are vetted and fully trustworthy.
+
+As administrator you have access to full information about the system through the `setup/system_info` policy, and also to user data, role editing, and many other critical aspects.
+
+The users in your organization who have backend access must be kept up-to-date.
+Any user leaving the organization must be disabled without delay.
+If a user takes on a new role in the organization, any required role changes for them in [[= product_name =]] must also be made as soon as possible.
+
+### Strong passwords
+
+Enforce strong passwords for all users.
+This is specially important for admin accounts and other privileged users.
+
+- Never go online with admin password set to `publish` or any other default value.
+- Introduce password quality checks. Make sure the checks are strict enough (length/complexity).
+- 16 characters is a quite secure minimum length. Don't go below 10.
+- If using [[= product_name =]] v4.5 or newer, enable the password rule that rejects any password which has been exposed in a public breach.
+
+!!! tip "Password rules"
+
+    See [setting up password rules](passwords.md#password-rules).
+
+### Protect against brute force attacks
+
+Consider introducing a measure against brute force login attacks, like CAPTCHA.
+Adjust timeout limits to your needs:
+
+When using the "forgot password" feature, a token is created which expires if the user doesn't click the password reset link that gets mailed to them in time.
+The time before it expires is set in the parameter `ibexa.site_access.config.default.security.token_interval_spec`.
```

Comment:
@glye I'm confused with this one.

- https://github.com/ezsystems/ezplatform-user/blob/v2.3.12/src/bundle/Resources/config/ezplatform_default_settings.yaml#L3 looks like the one I'm looking for.
- https://github.com/ezsystems/ezplatform-user/blob/v2.3.12/src/bundle/DependencyInjection/Configuration/Parser/Security.php#L20-L24 looks outdated
- https://doc.ibexa.co/en/3.3/release_notes/ez_platform_v3.0_deprecations/#event-names-changed has it as renamed JS event name?!?

```suggestion
The time before it expires is set in the parameter `ezsettings.default.security.token_interval_spec`.
```